nslord/Dockerfile

```diff
@@ -19,9 +20,11 @@ RUN echo 'deb [arch=amd64] http://repo.powerdns.com/ubuntu bionic-auth-46 main'
 RUN set -ex \
 	&& apt-key adv --keyserver keyserver.ubuntu.com --recv 0x1B0C6205FD380FBB \
 	&& apt-get update \
-	&& apt-get install -y pdns-server pdns-backend-mysql \
+	&& apt-get install -y pdns-server pdns-backend-mysql pdns-backend-pgsql postgresql-client-10 \
 	# credentials management via envsubst
 	&& apt-get -y install gettext-base \
+	# VPN route
+	&& apt-get -y install iproute2 \
```

Comment:
I don't think that's the right approach. We should first install the common stuff, and further down the non-common stuff, instead of installing everything everywhere.

Also, multi-stage builds may help (but I'm not sure).

Reply:
This deserves some more discussion. I've excluded this commit from this PR and moved it to https://github.com/desec-io/desec-stack/pull/633.